test/src/device-request-prompt.spec.ts

```diff
@@ -57,5 +57,5 @@ describe('device request prompt', function () {
         timeout: 1,
       })
     ).rejects.toThrow(TimeoutError);
-  });
+  }).timeout(1_000);
```

Comment:
Prefer to set it in the test itself as first line. That way its easier to understand 
```
this.timeout(1_000)
```